Framework/Algorithms/test/ExtractMaskToTableTest.h

```diff
@@ -71,7 +72,10 @@ class ExtractMaskToTableTest : public CxxTest::TestSuite {
 
     // Case: B has a larger range than A
     vecA.clear();
+    vecA.reserve(10);
+
     vecB.clear();
+    vecB.reserve(13);
```

Comment:
I would remove all those reserves. Some of them are useless since the capacity is already higher due to previous operations with the vectors. The others probably have completely negligible effect, I think unless you add huge amounts of data `std::vector` is very good at dealing with things itself, e.g, by overallocating.